commons/all/src/main/java/org/infinispan/commons/util/ImmutableIntSet.java

```diff
@@ -12,13 +12,11 @@
  *
  * @author Dan Berindei
  * @since 9.2
- * @deprecated since 9.3 This class will no longer be public, please use {@link IntSets#immutableSet(IntSet)}
  */
-@Deprecated
-public class ImmutableIntSet extends AbstractImmutableIntSet {
+class ImmutableIntSet extends AbstractImmutableIntSet {
```

Comment:
The new OffsetIntSet can also be made package private. It was just added in 15.0, so no worries about deprecation.

Reply:
Done